Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -114,7 +129,7 @@ def generate_csv(user, results)
           counts.values.sum,
         ]
       end
-      csv << [user, "*", "*", "*", "*", total(results)]
+      csv << [user, "*", "*", "*", "*", grand_total]
```

Comment:
Might be nice to have each of the `count` columns have values here:

```suggestion
      csv << [user, "*", "commits_total", "coauthorships_total", "signoffs_total", grand_total]
```

Doesn't need to block this PR, though!

Reply:
Done! Output now (for a single user):

```csv
user,repo,commits,coauthorships,signoffs,total
issyl0,brew,363,13,0,376
issyl0,core,473,24,327,824
issyl0,cask,4,0,0,4
issyl0,aliases,0,0,0,0
issyl0,autoupdate,1,0,0,1
issyl0,bundle,17,2,0,19
issyl0,command-not-found,1,0,0,1
issyl0,test-bot,3,0,0,3
issyl0,services,9,0,0,9
issyl0,cask-drivers,0,0,0,0
issyl0,cask-fonts,0,0,0,0
issyl0,cask-versions,0,0,0,0
issyl0,all,871,39,327,1237
```

Reply:
Yay, thanks @issyl0!